rosidl_typesupport_opensplice_c/resource/msg__type_support_c.cpp.em

```diff
@@ -61,11 +61,6 @@ extern "C"
 {
 #endif
 
-// Forward declare the get type support function for this type.
-ROSIDL_GENERATOR_C_EXPORT_@(spec.base_type.pkg_name)
-const rosidl_message_type_support_t *
-  ROSIDL_GET_TYPE_SUPPORT_FUNCTION(@(pkg), @(subfolder), @(msg))();
-
```

Comment:
I thought that would get pulled in by `rosidl_generator_c/message_type_support.h`:

https://github.com/ros2/rmw_opensplice/pull/139/files/938fb9a33a5783053cc1053c625596014f53e39b#diff-eb52319d09d2b6415f95c504f6609ba7L29

But now I'm not so sure, I was wrong when I initially thought this got rid of all warnings.